app/utils/buildSafeWallet.ts

```diff
@@ -0,0 +1,23 @@
+import { SafeConnector } from '@gnosis.pm/safe-apps-wagmi'
```

Comment:
This package is quite large: https://bundlephobia.com/package/@gnosis.pm/safe-apps-wagmi@1.0.3

Is it possible to do a dynamic import to reduce our bundle size? Not sure if rainbowkit supports an async `createConnector` method

Reply:
Not sure, as I can see rainbowkit `createConnector ` method should be synchronous according to types.